src/Core/Localization/Currency/DataLayer/CurrencyDatabase.php

```diff
@@ -122,23 +121,6 @@ protected function doRead($currencyDataId)
      */
     protected function doWrite($currencyDataId, $currencyData)
     {
-        if (!$currencyDataId instanceof LocalizedCurrencyId) {
-            throw new LocalizationException('First parameter must be an instance of ' . LocalizedCurrencyId::class);
-        }
-
-        $currencyCode = $currencyDataId->getCurrencyCode();
-        $currencyEntity = $this->dataProvider->getCurrencyByIsoCodeOrCreate($currencyCode, $currencyDataId->getLocaleCode());
-
-        $currencyEntity->iso_code = $currencyData->getIsoCode();
-        $currencyEntity->name = $currencyData->getNames()[$currencyDataId->getLocaleCode()];
-        $currencyEntity->numeric_iso_code = $currencyData->getNumericIsoCode();
-        $currencyEntity->symbol = $currencyData->getSymbols()[$currencyDataId->getLocaleCode()];
-        $currencyEntity->precision = $currencyData->getPrecision();
-
-        try {
-            $this->dataProvider->saveCurrency($currencyEntity);
-        } catch (Exception $e) {
-            throw new DataLayerException('Unable to persist data in DB data layer', 0, $e);
-        }
+        // Nothing.
```

Comment:
It would be nice to have a better explanation why there is `Nothing.`. :stuck_out_tongue: